packages/nextjs-kit/src/__tests__/data/exampleRecipeData.json

```diff
@@ -0,0 +1,27 @@
+[
+	{
+		"title": "Guacamole",
+		"field_ingredients": [
+			"1 Avocado",
+			"1/2 tomato",
+			"1/2 onion",
+			"salt",
+			"lime",
+			"black pepper",
+			"cilantro"
+		],
+		"field_media_image": {
+			"uri": {
+				"url": "https://dev-decoupled-drupal-qa.pantheonsite.io/sites/default/files/2022-07/guacamole.jpeg"
+			}
+		},
```

Comment:
There should be alt text available here as well. See https://github.com/pantheon-systems/decoupled-kit-js/blob/canary/starters/next-drupal-starter/__tests__/data/umamiEnRecipesData.json#L146 for the full shape of the `field_media_image` object.